messaging/src/test/java/org/axonframework/serialization/json/JacksonSerializerTest.java

```diff
@@ -103,7 +97,10 @@ void testSerializeAndDeserializeArray() {
 
     @Test
     void testSerializeAndDeserializeList() {
-        objectMapper.enableDefaultTyping(ObjectMapper.DefaultTyping.OBJECT_AND_NON_CONCRETE, JsonTypeInfo.As.PROPERTY);
```

Comment:
Note that change makes Axon incompatible with Jackson 2.9. Preferably, we would fall back to using `enableDefaultTyping` for pre-Jackson 2.10 versions.

Reply:
Bummer, of course, this isn't backwards compatible. I'll move to the old format prior to Jackson 2.10